README.md

```diff
@@ -275,7 +274,7 @@ pipeline {
 In scripted pipelines, wrap the relevant section with the `datadog` step like so:
 
 ```groovy
-datadog(testOptimization: [ enabled: true, serviceName: "my-service", languages: ["JAVA"], additionalVariables: [:] ]) {
+datadog(testOptimization: [ enabled: true, languages: ["JAVA"], additionalVariables: [:] ]) {
```

Comment:
In previous versions of the plugin, that property was mandatory?
Should we include something to specify that?

Reply:
I would rather leave it like this tbh, as I don't want to overcomplicate the docs. This feature doesn't get a lot of use, and I expect most of the people enabling Test Optimization to be those who have just installed a recent version of the plugin. If there's a customer issue I will ask for a plugin update